.circleci/config.yml

```diff
@@ -0,0 +1,118 @@
+version: 2.0
+
+jobs:
+  checkout:
+    docker:
+      - image: circleci/ruby:2-node-browsers
```

Comment:
Why ruby?

Reply:
It's one of circle ci's ready to use images that got the requirements we need installed and spin up fast

Reply:
You could probably use any of those with `node-browsers` on it
https://circleci.com/docs/2.0/circleci-images/

Reply:
Ah, I see, will switch it later